pkgs/development/php-packages/mongodb/default.nix

```
@@ -1,20 +1,21 @@
-{ buildPecl, lib, pkgs, pcre' }:
+{ stdenv, buildPecl, lib, pcre', pkg-config, cyrus_sasl, icu64
+, openssl, snappy, zlib, darwin }:
 
 buildPecl {
   pname = "mongodb";
 
   version = "1.9.0";
   sha256 = "16mbw3p80qxsj86nmjbfch8wv6jaq8wbz4rlpmixvhj9nwbp37hs";
 
-  nativeBuildInputs = [ pkgs.pkg-config ];
-  buildInputs = with pkgs; [
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [
     cyrus_sasl
     icu64
     openssl
     snappy
     zlib
     pcre'
-  ] ++ lib.optional (pkgs.stdenv.isDarwin) pkgs.darwin.apple_sdk.frameworks.Security;
+  ] ++ lib.optionals (stdenv.isDarwin) darwin.apple_sdk.frameworks.Security;
```

```suggestion
  ] ++ lib.optionals stdenv.isDarwin darwin.apple_sdk.frameworks.Security;
```
Also darwin should probably be dropped and replaced with Security. Something for another PR though.